content/en/docs/concepts/security/pod-security-standards.md

```diff
@@ -381,7 +381,7 @@ fail validation.
 		<tr>
   			<td style="white-space: nowrap">Seccomp (v1.19+)</td>
   			<td>
-  				<p>Seccomp profile must be explicitly set to one of the allowed values. Both the <code>Unconfined</code> profile and the <em>absence</em> of a profile are prohibited.</p>
+  				<p>On Linux, require that the seccomp profile is explicitly set to one of the allowed values. Both the <code>Unconfined</code> profile and the <em>absence</em> of a profile are prohibited.  This control only applies when you set `spec.os` to <code>linux</code>, or if you do not specify a Pod OS. Versions of Kubernetes before v1.25 enforce this control regardless of operating system.</p>
```

Comment:
> ... when you set `spec.os` to `linux` ...

It's actually coded the other way - skip the check when it's set to `windows`. A subtle difference, but it could matter if we added another OS.

Reply:
I wonder if I should also file a feature request issue to invert the logic there?

Reply:
exempting only for an explicitly windows pod is intentional, so we default in the safe direction for new hitherto unknown OSes